src/System.Runtime.Loader/tests/Configurations.props

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <BuildConfigurations>
       netcoreapp;
+      uap;
```

Comment:
@joperezr do you know if we have different configuration for uap-oat than uap? I mean is there a way to build a project as uap and build it differently for uap-oat?